app/views/catalog/_bulk_index_table.html.erb

```diff
@@ -21,14 +21,23 @@
 	    <td><%= job['argo.bulk_metadata.bulk_log_record_count'] %></td>
 	    <td><%= job['argo.bulk_metadata.bulk_log_druids_loaded'] %></td>
 
-	    <% druid_and_time = job['dir'].split(/\//) %>
-	    <td><%= link_to('Log', bulk_jobs_log_path(druid_and_time[0], druid_and_time[1]), data: { ajax_modal: 'trigger', ajax_modal_title: 'MODS Bulk Load Log' }) %></td>
-	    <td><%= link_to('XML', bulk_jobs_xml_path(druid_and_time[0], druid_and_time[1])) %></td>
+      <% if(job.has_key?('dir')) %>
```

Comment:
ugh, yeah, this looked ok in my text editor, but it's a hodgepodge of tabs and spaces, as it seems is often the case in view templates.  and actually, the second newly added if block should be indented another level regardless, now that i look at it.

will just add a commit fixing spacing.